documentation/URL_SPEC.md

```diff
@@ -0,0 +1,159 @@
+| `isa`    | Isosurface alpha, the opacity of the isosurface.                                          | Number in the range `[0, 1]`                                                                   | `1.0`                                  | `?c0=isa:1.0`                       |
+| `cpe`    | Use and show the control points instead of the ramp on load.                              | `1` (enabled) or `0` (disabled)                                                                | `0`                                    | `?c0=cpe:1`                         |
+| `cps`    | Control points for the transfer function. If provided, overrides the `lut` field.         | List of `bin_index:opacity:color` triplets, separated by a colon ([_see 'Binning'_](#binning)) | `0:0:ffffff:255:1:ffffff`              | `?c0=cps:0:0:ff0000:150:0.5:ffff00` |
+| `rmp`    | Raw ramp values. If provided, overrides the `lut` field when calculating the ramp values. | Two float bin indices, formatted as `min:max` ([_see 'Binning'_](#binning))                    | `0:255`                                | `?c0=rmp:0:255`                     |
+| `lut`    | Lookup table (LUT) to map from volume intensity values to opacity.                        | [_See 'Lookup Table'_](#lookup-table-lut)                                                      | `0:255`                                | `?c0=lut:0:255`                     |
+
+### Binning
+
+When loaded, each channel's raw intensity values are sorted into one of 256
+bins, where the bin index `0` holds the minimum raw intensity value of the
+channel and the bin index `255` holds the maximum raw intensity value. Certain
+properties like `cps` (control points), `rmp` (ramp), and `lut` (lookup table)
+directly reference these bin indices.
+
+![Screenshot of Volume Viewer with a volume in advanced mode. A histogram with
+255 bins is visible, with a ramp increasing from 0 to 1 on the Y axis in the
+span of 53 to 136 on the X axis.](./assets/example_histogram.png)
+
+For example, the above control points are at bin indices `0`, `53`, `136`, and
+`255`. Depending on the range of the channel's values, these may
+represent different raw intensity values.
+
+| Min raw intensity | Max raw intensity | Bin `0` intensity | Bin `53` intensity | Bin `136` intensity | Bin `255` intensity |
+| ----------------- | ----------------- | ----------------- | ------------------ | ------------------- | ------------------- |
+| 0                 | 255               | 0                 | 53                 | 136                 | 255                 |
+| 50                | 200               | 50                | 103                | 166                 | 200                 |
+| 0                 | 1                 | 0                 | 0.207              | 0.533               | 1                   |
```

Comment:
@frasercl @toloudis is this an accurate description of how the binning works?

Reply:
You might want to say something to remind users that the bins represent a range of values (like any binned histogram).  So if the range of intensities is greater than 256, then the bins span over multiple intensities.  Pixels in an intensity range go into  the bins

Reply:
Changed the last row to have a domain of greater than 256 values, and updated the bins to be ranges.